packages/discovery-react-components/src/components/SearchFacets/utils/buildAggregationQuery.ts

```diff
@@ -9,7 +9,18 @@ export const buildAggregationQuery = (configuration: QueryAggregationWithName[])
         let nestedTypeTermAgg = '';
         if (field.includes('enriched_') && field.includes('entities.text')) {
           const topLevelTermEntityField = field.split('.')[0];
+          const topLevelNestedField = field.slice(0, field.lastIndexOf('.'));
           nestedTypeTermAgg = `.term(${topLevelTermEntityField}.entities.type,count:1)`;
+          return (
+            'nested(' +
+            topLevelNestedField +
+            ').term(' +
+            field +
+            termCount +
+            termName +
+            ')' +
+            nestedTypeTermAgg
+          );
```

Comment:
I think this would be easier to read with string interpolation:
```suggestion
          return (`nested(${topLevelNestedField}).term(${field}${termCount}${termName})${nestedTypeTermAgg});
```